docs/manual/intro-to-boa.md

```diff
@@ -131,6 +131,32 @@ bytes.fromhex(foobar.toString('hex'));
 // "b'foobar'"
 ```
 
+### Use of Generator
+
+`Boa` will also handle `Generator` class in the native python runtime. JavaScript syntax of `Generator` could be applied to Python `Generator` directly:
+
+```js
+
+const generator = generatorFromPython(); // Generator
```

Comment:
I think it's better to represent it from Python:

- prepend a python block to write a generator function.
- import the generator object using boa.import here.